fsharp.html.markdown

```diff
@@ -53,8 +53,9 @@ add 2 3                       // Now run the function.
 
 // to define a multiline function, just use indents. No semicolons needed.
 let evens list =
-   let isEven x = x % 2 = 0     // Define "isEven" as a sub function
-   List.filter isEven list    // List.filter is a library function
+   let isEven x = x % 2 = 0   // Define "isEven" as a sub function. Note
+   List.filter isEven list    // that equality operator is single char "=".
+                              // List.filter is a library function
                               // with two parameters: a boolean function
```

Comment:
Could you please change this to
```
let evens list =
   let isEven x = x % 2 = 0   // Define "isEven" as a sub function. Note
   						      // that equality operator is single char "=".
   List.filter isEven list    // List.filter is a library function
                              // with two parameters: a boolean function
```

I think this would make your change stand out more clearly.